.github/workflows/tuf_client_tests.yml

```diff
@@ -46,3 +46,7 @@ jobs:
             --root repository/repository/2.root.json \
             -t http://localhost:8001/targets \
             -m http://localhost:8001
+      # Test with python-tuf ngclient
```

Comment:
perfect! Can we merge this and ignore failure? https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#jobsjob_idstepscontinue-on-error

With a follow-up issue marked with milestone v5 to clean-up so we don't forget :)

Reply:
Added in [c3045d6](https://github.com/sigstore/root-signing/pull/343/commits/c3045d6815d837e9d11edeab4d91809bb5feaa46) and files https://github.com/sigstore/root-signing/issues/346 to make the updates